release_instructions.md

```diff
@@ -306,3 +306,38 @@ For checking that the build process is ongoing as expected:
        1. This pull request may take a minute or two to open.
        1. Once it is open, make a comment containing the text "build bottle".
           For further details, see the [README at osrf/homebrew-simulation](https://github.com/osrf/homebrew-simulation#to-build-bottles).
+
+    1. A pull request was opened to
+        <https://github.com/gazebo-release/$VENDOR_PACKAGE/>, where
+        `$VENDOR_PACKAGE` is the name of the Gazebo vendor package for ROS (see
+        <https://gazebosim.org/docs/latest/ros2_gz_vendor_pkgs/>)
+
+        1. Ensure that the ROS CI build passes. Note that some builds have
+            warnings which show up as failures on the Github UI while the build
+            actually succeeded.
+        1. For packages that are considered to be part of ROS core
+            (`gz_cmake_vendor`, `gz_utils_vendor`, and `gz_math_vendor`), start
+            additional CI tests using ci.ros2.org to build and test all ROS core
```

Comment:
can anyone start additional CI tests on ci.ros2.org or does this require additional permission?

Reply:
I've added a note about this.